##### Supplementary Information for 
## Bayesian phylogenetics on globally emerging SARS-CoV-2 variant BA.2.86 suggest global distribution and rapid evolution
#### Andrew P. Rothstein, Xueting Qiu, Keith Robison, Susan Collins, Gabi Muir, Bernadette Lu, Alex M. Plocik, Birgitte B. Simen, Casandra W. Philipson

Ginkgo Bioworks, 27 Drydock Ave 8th Floor, Boston, MA 02210, https://www.ginkgobioworks.com/

#### Abstract 
Using bioinformatic pipelines and Bayseian phylogenetic analyses, we characterized a SARS-CoV-2 variant designated by the World Health Organization as a variant under monitoring in August 2023. Here we analyze the genomes of this SARS-CoV-2 variant, BA.2.86, deposited into GISAID within the two weeks of its emergence (2023-08-14 first submission to 2023-08-31), including the first BA.2.86 genome reported from a traveler originating from Japan. We present bioinformatics methods using publicly available tools to help analysts identify the lineage-defining 12 nucleotide insertion (S:Ins16MPLF), which is often masked by most bioinformatics pipelines. We also applied maximum-likelihood and Bayesian phylogenetics to demonstrate the high mutational rate of the tree branch leading to the emergence of BA.2.86, hinting at possible origins, and predict that BA.2.86 emerged around May 2023 and spread globally rapidly. Taken together, these results provide a framework for more rigorous bioinformatics approaches for teams performing genomic surveillance on viral respiratory pathogens.
